README.md

```diff
@@ -1 +1,32 @@
-# Awesome-Books-ES6
\ No newline at end of file
+# Awesome-Books-ES6
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome_books
+
+> This project is the skeleton for a book management application.
+
+
+## 🧰 Built With
+
+- HTML
+- JS
+
+## Author
+
+👤 **Matias Aguirre**
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] This link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯